docs/source/query/ept_stores.ipynb

```diff
@@ -20,29 +22,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pdal\n",
-    "import json\n",
-    "import os\n",
-    "import folium\n",
-    "import geopandas as gpd\n",
-    "from pyproj import CRS, Transformer\n",
-    "import datetime\n",
-    "import gpstime"
+    "#!pip install folium geopandas gpstime pyproj"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "# Querying ATL08 Entwine Point Tiles\n",
+    "import json\n",
+    "import os\n",
+    "import datetime\n",
     "\n",
-    "This notebook provides sample code for querying and visualizing the Entwine Point Tiles Store for ATL08 V003.\n",
+    "import gpstime\n",
+    "from pyproj import CRS, Transformer\n",
+    "import folium\n",
+    "import geopandas as gpd\n",
+    "#import pdal\n",
     "\n",
-    "[Entwine Point Tiles (EPT)](https://entwine.io/entwine-point-tile.html) are a new \"cloud-optimized\" storage format for large point cloud data."
+    "from maap.maap import MAAP\n",
+    "maap = MAAP(maap_host='api.ops.maap-project.org')"
```

Comment:
Is there a reason why this is `api.ops.maap-project` instead of just `api.maap-project` ?

Reply:
https://github.com/MAAP-Project/ZenHub/issues/367